gridstatus/lmp_config.py

```diff
@@ -0,0 +1,154 @@
+import datetime
+import functools
+import inspect
+
+import pandas as pd
+
+from gridstatus.base import ISOBase, Markets, NotSupported
+
+
+class lmp_config:
+
+    configs = {}
+
+    def __init__(self, supports, tz=None):
+        self.supports = supports
+        self.tz = tz
```

Comment:
do we need this `tz` parameter at all? not seeing it used anywhere.

Reply:
Whoops